app/controllers/questions_controller.rb

```diff
@@ -22,6 +22,7 @@ def create
 
   def show
     @question = Question.find(params[:id])
+    @answer = @question.answers.new
```

Comment:
Is there a way we could make this into a presenter? I have started one in the models folder if you would like to add to it or create one specific for your controller. @TheKaterTot @Ryanspink1